lib/internal/crypto/random.js

```diff
@@ -119,6 +120,76 @@ function randomFill(buf, offset, size, cb) {
   _randomBytes(buf, offset, size, wrap);
 }
 
+// Largest integer we can read from a buffer.
+// e.g.: Buffer.from("ff".repeat(6), "hex").readUIntBE(0, 6);
+const RAND_MAX = 281474976710655;
```

Comment:
```suggestion
const RAND_MAX = 0xFFFF_FFFF_FFFF;
```
Arguably looks a little more Obviously Correct when written that way.